prediction_market_agent_tooling/agent/development_tools/test_single_agent.py

```
@@ -0,0 +1,163 @@
+        use_old_research=use_old_research,
+        use_old_prediction=use_old_prediction,
+        max_trades_to_test_on=max_trades_to_test_on,
+        run_name=f"test_{agent_name}",
+        mocked_agent_name=agent_name,
+        delay_between_trades=delay_between_trades,
+    )
+
+    test_results = tester.test_prophet_agent(
+        agent_data, research_agent, prediction_agent
+    )
+    evaluation_metrics = tester.evaluate_results(
```

Ruff (0.11.9)

Local variable `evaluation_metrics` is assigned to but never used

Remove assignment to unused variable `evaluation_metrics`

(F841)